src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java

```diff
@@ -297,6 +296,11 @@ public void mouseMoved(EditorMouseEvent e) {
         }
     }
 
+    private boolean isPlainTextLanguageFile(PsiFile file) {
```

Comment:
```suggestion
    private boolean isSupportedLanguageFile(PsiFile file) {
```

Reply:
@gayanper WDYT since now we check for the registered server definitons as well.

Reply:
I think thats a good suggestion. I made the change and pushed it